src/transformers/models/encoder_decoder/modeling_tf_encoder_decoder.py

```diff
@@ -569,13 +569,12 @@ def call(
                 "output_hidden_states": output_hidden_states,
                 "return_dict": return_dict,
                 "training": training,
-                "kwargs_call": kwargs_encoder,
+                "kwargs_call": {}
             }
 
             # Add arguments to encoder from `kwargs_encoder`
             for k, v in kwargs_encoder.items():
                 encoder_processing_inputs[k] = v
```

Comment:
You can see here that all contents in `kwargs_encoder` are dumped into `encoder_processing_inputs`, so they were effectively being passed twice.

`input_processing` expects `kwargs_encoder` to be empty, except under very special circumstances (when using deprecated arguments)